src/sentry/models/apiscopes.py

```diff
@@ -31,13 +31,16 @@ class ApiScopes(Sequence):
 
     member = (("member:read"), ("member:write"), ("member:admin"))
```

Comment:
👉 👈   do you mind adding the [member:invite](https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L1892) scope here while you're at it 🥺